Icon conversion: support GraphicsMagick in addition to  ImageMagick (#1002)
nativefier/nativefier: docs/api.md

The icon parameter can either be a `.icns` or a `.png` file if the [optional dependencies](../README.md#optional-dependencies) are installed.

> If you have the optional dependencies - `iconutil`; and Imagemagick `convert` and `identify`, or GraphicsMagic `gm` - in your `PATH`, Nativefier will automatically convert the `.png` to a `.icns` for you.

Reviewer: ```suggestion
If your `PATH` has our image-conversion dependencies (`iconutil`, and either ImageMagick `convert` + `identify`, or GraphicsMagick `gm`), Nativefier will automatically convert the `.png` to a `.icns` for you.
```
